.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.git
```

Comment:
It slipped. :innocent:
Documentation [is here](https://docs.docker.com/reference/builder/#dockerignore-file), in short it speeds up docker image build by not copying the ~50M of git and node modules stuff.